test/feature-benchmark/mzcompose.py

```diff
@@ -393,7 +389,7 @@ def workflow_default(c: Composition, parser: WorkflowArgumentParser) -> None:
     scenarios_to_run = [
         scenario
         for i, scenario in enumerate(selected_scenarios)
-        if shard is None or shard_count is None or i % int(shard_count) == shard
+        if buildkite.accepted_by_shard(i, shard, shard_count)
```

Comment:
Sorry I am late for the party, but for the record, only the first argument is needed here -- the two others can be obtained internally by the `accepted_by_shard` method. This will remove individual calls to `get_parallelism_index` and `get_paralellism_count` in every mzcompose.

Reply:
Yeah, this is a good suggestion! Will do this in a follow-up!